### PR TITLE
Unread Dot

### LINKED ISSUE #96: [Feature Request] Daily Messages Unread Dot
**Describe the solution you'd like**

Display a dot on the menu icon when there are unread info center messages available.

This will work by saving the size of the list to the shared preferences every time you open the info center. Then every time the timetable is loaded it will also load the messages and check if the list size is greater than the saved size. If so it will display the dot to inform the user that there are new unread messages in the info center.

### PR DESCRIPTION
**Pull request by Domi04151309**
Closes #96

### CODE REVIEW

File: app/src/main/res/drawable/all_infocenter_dot.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:fillColor="?android:attr/textColorPrimary"
+      android:pathData="M17.912,1.002A3,3 0,1 0,21 4a3,3 0,0 0,-3.088 -2.998zM6,2c-1.1,0 -2,0.9 -2,2v16c0,1.1 0.9,2 2,2h12c1.1,0 2,-0.9 2,-2L20,8.578a5,5 0,0 1,-2 0.418L18,20L6,20L6,4h1v9l3,-2.25L13,13L13,4a5,5 0,0 1,0.424 -2L6,2zM9,4h2v5l-1,-0.75L9,9L9,4z"/>
```
Comment (SapuSeven) [inline app/src/main/res/drawable/all_infocenter_dot.xml:8]: What would you think about coloring the dot red to make it stand out more?

```suggestion
    <path
        android:fillColor="?android:attr/textColorPrimary"
        android:pathData="M6,2c-1.1,0 -2,0.9 -2,2v16c0,1.1 0.9,2 2,2h12c1.1,0 2,-0.9 2,-2L20,8.578a5,5 0,0 1,-2 0.418L18,20L6,20L6,4h1v9l3,-2.25L13,13L13,4a5,5 0,0 1,0.424 -2L6,2zM9,4h2v5l-1,-0.75L9,9L9,4z" />
    <path
        android:fillColor="#FFF44336"
        android:pathData="M17.912,1.002A3,3 0,1 0,21 4a3,3 0,0 0,-3.088 -2.998z" />
```

Comment (Domi04151309) [reply app/src/main/res/drawable/all_infocenter_dot.xml:8]: That wouldn't work because the icons get tinted.

Comment (SapuSeven) [reply app/src/main/res/drawable/all_infocenter_dot.xml:8]: I see. That's unfortunate.

**Comment by Domi04151309** [comment]
What's the status of this PR?

**Review by SapuSeven — DISMISSED** [review]
It's not perfect, but it will work for now.

**Review by SapuSeven — APPROVED** [review]

**Pull request merged**